packages/language-server/src/server.ts

```diff
@@ -11,6 +11,8 @@ import {
   CodeActionKind,
   CodeActionParams,
 } from 'vscode-languageserver'
+import * as checkpoint from 'checkpoint-client'
```

Comment:
Nit: maybe we should not import the whole library if we are just using a single function from it 🤷

Reply:
Good catch - changed that

Reply:
Good catch - changed that